lib/node.js

```diff
@@ -271,14 +269,19 @@ Node.prototype._syncBitcoind = function() {
       }
     });
   }, function(err) {
-    log.info('Stopping Bitcoind Sync');
+
     self.bitcoindSyncing = false;
     self.chain.lastSavedMetadataThreshold = 0;
     if (err) {
       Error.captureStackTrace(err);
       return self.emit('error', err);
     }
-    self.emit('synced');
+
+    // If bitcoind is completely synced
+    if (self.bitcoind.isSynced()) {
+      self.emit('synced');
+    }
+
```

Comment:
The synced event can be emitted more than once, which we don't want:

```
  // Notify that there is a new tip
  this.bitcoind.on('tip', function(height) {
    log.info('Bitcoin Core Daemon New Height:', height);
    self.bitcoindHeight = height;
    self._syncBitcoind();
  });
```

You should keep track somewhere that you emitted a synced event and not emit it again.